cmd/genesis/newgenesis.go

```diff
@@ -70,6 +61,12 @@ func main() {
 	if !*quiet {
 		verboseOut = os.Stdout
 	}
+
+	if *short {
+		genesisData.FirstPartKeyRound = 0
```

Comment:
I prefer to keep the flag to limit LastPartKeyRound.
That is, not to change FirstPartKeyRound , and change LastPartKeyRound  only if it is more than 1500.